eslint.config.js

```diff
@@ -0,0 +1,18 @@
+import typescriptEslint from 'typescript-eslint'
+
+import { config } from './lib/index.js'
+
+/** @type {import('eslint').Linter.Config[]} */
+export default typescriptEslint.config(
```

Comment:
Can't we use imported config directly, what makes this package different from other packages ?